lib/EndOfItem/EndOfItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Layout,
+  Icon,
+} from '@folio/stripes/components';
+
+import css from './EndOfItem.css';
+
+const EndOfItem = memo(({
+  title = <FormattedMessage id="stripes-components.endOfList" />,
+  className,
+}) => (
+  <Layout className={classnames('textCentered', css.endOfItem, className)}>
```

Comment:
I suggest removing css.endOfItem in favor of approach below.
```suggestion
  <Layout className={classnames('textCentered', 'padding-top-gutter', className)}>
```